### PR TITLE
feat(stale): Recency warning if Colophon file is stale

### PR DESCRIPTION
**Pull request by kspaans**
Addresses #15 and #19 

Requires a new column to be added to the `repositories` table. Also confirms #7 that the install instructions are accurate.

### CODE REVIEW

File: lib/graph.js [hunk 1]
```diff
@@ -75,6 +86,11 @@ exports.process = async repository => {
   // attempt to parse content
   try {
     repository.colophon = await parser(repository.yaml.content)
+    const date = new Date(repository.committedDate.blame.ranges[0].commit.committedDate)
+    const cutoffMillisecs = (new Date().getTime()) - (6 * 30 * (24 * 60 * 60 * 1000))
+    let cutoff = new Date()
+    cutoff.setTime(cutoffMillisecs)
+    repository.stale = date < cutoff
```
Comment (ahmadnassri) [inline lib/graph.js]: you're calculating the stale value at the time of the insert which for anything other than the initial scan will always be false 

e.g. all the webhooks

**Comment by kspaans** [comment]
Looks like I've misunderstood a bit, do we want the last time the `.colophon.yml` file was touched in the repo, or the last time the repo was checked by the Colophon app? I agree we already have the latter, and I've done the former. I'll go with the repo updated time, but have already done most of the work for pulling out the commit history of the `.colophon.yml` file.

**Comment by ahmadnassri** [comment]
your code pulls in the last updated time of the file, but is not storing that, it's storing the flag, so I'm saying if we store the last updated time of the file as the `updated` column, that's a good first step.

secondary: the rest of context I was adding about storing changes over time, but that can come later

File: database/app.sql [hunk 2]
```diff
@@ -24,9 +15,6 @@ ALTER TABLE repositories ADD COLUMN IF NOT EXISTS colophon jsonb;
 -- trigger to automatically update timestamp
 DROP TRIGGER IF EXISTS set_timestamp ON repositories;
```
Comment (ahmadnassri) [inline database/app.sql]: need to remove this as well

File: lib/db/repository.js [hunk 3]
```diff
@@ -22,12 +22,13 @@ exports.add = (installation, repo) => {
     repo.private,
     repo.filename,
     repo.content,
-    repo.colophon
+    repo.colophon,
+    repo.updated
```
Comment (ahmadnassri) [inline lib/db/repository.js]: did you confirm the date format here is the same as what PSQL expects?

Comment (kspaans) [reply lib/db/repository.js]: Good catch! I did not! It looked right in my local testing, but I'll doublecheck the format returned by the GitHub API, and what PSQL expects.

Comment (kspaans) [reply lib/db/repository.js]: Confirmed! The GraphQL type returned is [ISO-8601](https://developer.github.com/v4/scalar/datetime/) which is also one of the formats accepted by PSQL. :chart_with_upwards_trend:

**Pull request merged**